openslides/assignments/static/js/assignments/site.js

```diff
@@ -753,11 +705,12 @@ angular.module('OpenSlidesApp.assignments.site', [
     '$filter',
     'gettextCatalog',
     'AssignmentPoll',
-    'assignmentpoll',
+    'assignmentpollId',
     'ballot',
-    function($scope, $filter, gettextCatalog, AssignmentPoll, assignmentpoll, ballot) {
+    function($scope, $filter, gettextCatalog, AssignmentPoll, assignmentpollId, ballot) {
         // set initial values for form model by create deep copy of assignmentpoll object
         // so detail view is not updated while editing poll
+        var assignmentpoll = AssignmentPoll.get(assignmentpollId);
```

Comment:
Is it necessary to use angular.copy here? Cf. other update controller in other apps.